CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+#### Contributing to a Manim Community Fork - General guidelines
+
+As this is a Fork of [Manim from 3b1b](https://github.com/3b1b/manim), contributing can be a bit confusing. So here is a small guide on it. 
+
+1. First, make a fork of either this Repo or the Repo from which this was Forked. ( 3b1b Repository)
```

Comment:
In the very near future, cloning from 3b1b is going to be far from ideal, so I'd just instruct folks to fork from us.